docs/updating-dependencies.md

```diff
@@ -0,0 +1,52 @@
+# EBS CSI Driver Dependencies Upgrade Guide
+
+## Pre-release dependencies checklist
+
+For code security and hygiene, it’s important to keep all of our dependencies up to date ahead of each EBS CSI Driver release.
```

Comment:
Replace the word `our` with `the`, repeat for every other instance of `our` in the file.